modules/openy_features/openy_node/modules/openy_node_landing/config/install/core.entity_view_display.node.landing_page.default.yml

```diff
@@ -53,6 +62,14 @@ content:
     third_party_settings: {  }
     type: list_default
     region: content
+  field_lp_location:
```

Comment:
Do we need it to be displayed here? I believe it should be hidden.

Reply:
It's default view mode where displayed all added to bundle fields. I don’t think that we should hide the location field when all other fields are displayed. And in fact, this mode is not used anywhere.